### PR TITLE
Develop to master

### PR DESCRIPTION
**Pull request by ThrawnCA**
- Make Google Analytics plugin and error pages recognise Flask
- Restore download headers on exported reports
- Use URLs instead of controller names to control template content
- Ensure our custom styles override core
- Add more scenario tests from ckanext-qgov

### CODE REVIEW

File: ckanext/data_qld/templates/header.html [hunk 1]
```diff
@@ -59,9 +59,6 @@
 </div>
 {% endblock %}
 <header class="navbar navbar-static-top masthead">
-  {% block header_debug %} {% if g.debug and not g.debug_supress_header %}
-  <div class="debug">Controller : {{ c.controller }}<br/>Action : {{ c.action }}</div>
```
Comment (duttonw) [inline ckanext/data_qld/templates/header.html:63]: why did we remove this?

Comment (duttonw) [inline ckanext/data_qld/templates/header.html:63]: why did we remove this?

**Review by duttonw — APPROVED** [review]

File: ckanext/data_qld/templates/header.html (hunk 1 above)
Comment (ThrawnCA) [reply ckanext/data_qld/templates/header.html:63]: It depends on a Pylons-specific feature and is not really needed anyway.

I could restore it and just call `super()` if you prefer?

Comment (duttonw) [reply ckanext/data_qld/templates/header.html:63]: go with super, we should try to be 'light' touch so that we should keep forward compatibility

**Review by duttonw — APPROVED** [review]

**Pull request merged**